src/Sylius/Bundle/CoreBundle/Behat/CoreContext.php

```diff
@@ -430,7 +430,16 @@ public function thereAreLocales(TableNode $table)
 
         foreach ($table->getHash() as $data) {
             $locale = $repository->createNew();
-            $locale->setCode($data['code']);
+
+            if (isset($data['code'])) {
+                $locale->setCode($data['code']);
+            } elseif (isset($data['name'])) {
+                $code = $this->getLocaleCodeByEnglishLocaleName($data['name']);
+
```

Comment:
I'd actually avoid the variable, it is used only once. Just set the result of method directly.